src/index.ts

```diff
@@ -75,11 +75,16 @@ export type EdgesOptionsSpec = OffsetLimitSpec & {
   direction?: Direction;
 };
 
+export type ColumnType = 'number' | 'label' | 'category' | 'date' | 'boolean';
```

Comment:
Should we also include interfaces for entity and table metadata, like it's done on the [server](https://github.com/multinet-app/multinet-server/blob/2cf8989b61bf3d9cf152e7df8cb462e41351f1d9/multinet/types.py#L12-L34)?

Reply:
After a discussion, we decided not to make further changes here due to #491.